apps/meteor/app/livechat/server/lib/QueueManager.ts

```diff
@@ -52,6 +55,7 @@ export const queueInquiry = async (inquiry: ILivechatInquiryRecord, defaultAgent
 		// We'll queue these inquiries so when new license is applied, they just start rolling again
 		// Minimizing disruption
 		await saveQueueInquiry(inquiry);
+		await QueueManager.dispatchInquiryPosition(inquiry);
```

Comment:
This should only happen when `settings.get('Livechat_waiting_queue')` is true (which is an EE setting)